tests/automated_tests/import_example_test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
please remove all duplicate headers

Reply:
Even if I remove it, pre-commit hook adds it again

Reply:
then you need to check the config of pre-commit

Reply:
That is known issue, I could only disable it for my files